study-builder/studies/rarex/i18n/en.json

```diff
@@ -255,13 +249,13 @@
   "general_information_hic_not_to_answer_details": "What type(s) of coverage do you have?",
   "general_information_hic_not_to_answer_details_exp": "Please describe the general type of insurance(s) you have.",
 
-  "general_information_patient_country_birthplace": "Country/Territory where the patient was born *",
+  "general_information_patient_country_birthplace": "Country/Territory where the patient was born*",
   "general_information_patient_country_birthplace_req_hint": "Country/Territory cannot be empty",
 
-  "general_information_patient_city_birthplace": "City/Municipality where the patient was born *",
+  "general_information_patient_city_birthplace": "Name of the city/municipality where the patient was born*",
   "general_information_patient_city_birthplace_req_hint": "City/Municipality cannot be empty",
 
-  "general_information_patient_state_birthplace": "State/Province (or other applicable region) where the patient was born *",
+  "general_information_patient_state_birthplace": "State where the patient was born*",
```

Comment:
This doesn't look right. It should be `State/Province (or other applicable region) ...`